Manito/Manito/Screens/Setting/SettingViewController+MailComposeViewControllerDelegate.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  SettingViewController+MailComposeViewControllerDelegate.swift
+//  Manito
+//
+//  Created by 이성호 on 2023/05/12.
+//
+
+import MessageUI
+
+// MARK: - MFMailComposeViewControllerDelegate
+extension SettingViewController: MFMailComposeViewControllerDelegate {
+    func sendReportMail() {
+        if MFMailComposeViewController.canSendMail() {
+            let composeVC = MFMailComposeViewController()
```

Comment:
p1;

<img width="388" alt="image" src="https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/assets/78677571/8ebb376e-9d19-44c8-a5b4-84b8099cd3c0">

```suggestion
            let composeViewController = MFMailComposeViewController()
```

하핫..